packages/components/src/FormField/hooks/useAtlantisReactHookForm.ts

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useImperativeHandle } from "react";
+import { useController, useForm, useFormContext } from "react-hook-form";
+import { mergeRefs } from "@jobber/components/utils/mergeRefs";
+import { FormFieldProps } from "../FormFieldTypes";
+
+interface useAtlantisReactFormProps
+  extends Pick<
+    FormFieldProps,
+    "actionsRef" | "defaultValue" | "value" | "validations" | "inputRef"
+  > {
+  name: string;
+}
+
+/**
+ * Hook used to manage the form state of a field through react-hook-form
+ */
+export function useAtlantisReactForm({
```

Comment:
The name of the hook and the name of the file are slightly different. I think I have a slight preference for including "hook" in the name because this implementation is specific to RHF and then the hook name makes that clear.

Reply:
Thanks for catching that, I had done a lot of renaming and must have missed this

Reply:
Updated in [64c0733](https://github.com/GetJobber/atlantis/pull/2184/commits/64c0733eaef732b61fdae4a38b858058ef051382)